jetstream/test_tas_api.py

```diff
@@ -17,15 +16,15 @@
 )
 
 
+@modify_settings(INSTALLED_APPS={
+    'append': 'jetstream',
+})
 class TestJetstream(TestCase):
     """Tests for Jetstream allocation source API"""
 
-    def setUp(self):
-        if 'jetstream' not in settings.INSTALLED_APPS:
-            self.skipTest('jetstream not in settings.INSTALLED_APPS')
-
+    @override_settings(TACC_API_URL='https://localhost/api-test')
     @my_vcr.use_cassette()
-    def test_validate_account(self, cassette):
+    def test_validate_account(cassette, self):
```

Comment:
This is crazy. I don't know why the decorators messed up the order of the parameters. How do computers even work.